tests/test_compression.py

```diff
@@ -36,6 +36,11 @@
 ]
 
 
+def test_compression__does_not_raise_error_with_horizontal_line():
+    flat_data = np.array([list(range(100)), [10 for _ in range(100)]], dtype=np.int32)
+    compress_filtered_gmr(flat_data)
+
+
 def test_compression_performance(new_A1):
     # data creation, noise cancellation, peak detection
```

Comment:
I don't know how to make a comment about the PNG files...but do you know why they changed? I would have thought they'd be the same.  ...also, aren't they supposed to be git-ignored anyway? do you know why they're showing up in the repo?

Reply:
I think they may have been gitignored in the original codecommit but that may have been lost in the transfer to github

Reply:
That is likely what's happening. I'll remove them